modern/.env

```diff
@@ -1,5 +1,5 @@
-CONAN_VERSION=1.47.0
-DOCKER_TAG=1.47.0
+CONAN_VERSION=2.0.0-alpha6
+DOCKER_TAG=2.0.0-pre
```

Comment:
Here we want latest `1.x`. Remember to change this before merging PR

Reply:
done